projects/manage-supervision-and-delius/src/main/kotlin/uk/gov/justice/digital/hmpps/integrations/delius/user/entity/User.kt

```diff
@@ -29,6 +30,23 @@ class User(
     val surname: String,
 )
 
+interface UserRepository : JpaRepository<User, Long> {
+
+    @Query(
+        """
+        select u
+        from User u
+        join fetch u.staff s
+        join fetch s.provider p
+        where u.username = :username
```

Comment:
Username is case-insensitive in Delius, and there's an index on `upper(username)`.
```suggestion
        where upper(u.username) = upper(:username)
```